account_statement_import_online/models/online_bank_statement_provider.py

```diff
@@ -93,7 +92,6 @@ class OnlineBankStatementProvider(models.Model):
     certificate_public_key = fields.Text()
     certificate_private_key = fields.Text()
     certificate_chain = fields.Text()
-    allow_empty_statements = fields.Boolean()
```

Comment:
I was reading the comments and story of this PR, and I can't figure out why this field is being deleted. 
@victoralmau  do you remember this PR? Could you explain to me why this change is made?

Reply:
@stefan-tecnativa Empty statements where a bit strange always, but if you wanted to have a statement for each day, they could be used to check for completeness: have we retrieved all the statements or are there gaps. In 16.0 statement date has become a computed field, dependent on the statement lines. So this does not work anymore. In a lot of ways in 16.0 a statement is a kind of summary of the lines it contains. So empty statements do not make sense and should no longer be created or even allowed.

Reply:
@NL66278 thank you very much for the explanation!